examples/mobile/UIComponents/components/list/basiclist.html

```diff
@@ -19,16 +19,41 @@ <h1>
 		<div class="ui-content">
 			<ul class="ui-listview ui-content-area">
 				<li>
-					Main text 01
+					<div class="ui-li-text">
+						<span class="ui-li-text-title">
+							Main text 01
+						</span>
+						<span class="ui-li-text-sub ui-li-text-value">
+							Value text
+						</span>
+					</div>
 				</li>
+				<div class="ui-li-divider"></div>
```

Comment:
1. This is wrong HTML syntax, `<ul>` shouldn't contains `div` elements.
2. I'm think we don't need "ui-li-divider", please use just "border-bottom" css property.

Reply:
We can use `ui-li-divider` between lists eg. 
```
<ul>
   <li></li>
</ul>
<div class="ui-li-divider"></div>
<ul>
   <li></li>
</ul>
```

Reply:
One correction. We cannot use border-bottom because the line is not on whole width of list item. We have to use pseudo element `::before` or `::after`.

Reply:
Okay, I removed the "div.ui-li-divider" and move it into the "li.ui-li-divider".